### PR TITLE
Add LeetCode and CodeSignal

### PR DESCRIPTION
**Pull request by aarontorres0**

### CODE REVIEW

File: entries/c/codesignal.com.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "CodeSignal": {
+        "domain": "codesignal.com",
+        "url": "https://codesignal.com",
+        "contact": {
+            "facebook": "codesignalcom",
+            "twitter": "CodeSignalCom"
+        },
+        "keywords": [
+            "education"
+        ]
```
Comment (phallobst) [inline entries/c/codesignal.com.json:13]:
```suggestion
        ],
        "regions": [
            "us"
        ]
```

File: entries/c/codesignal.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "CodeSignal": {
+        "domain": "codesignal.com",
+        "url": "https://codesignal.com",
```
Comment (phallobst) [inline entries/c/codesignal.com.json]:
```suggestion
```

File: entries/l/leetcode.com.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+    "LeetCode": {
+        "domain": "leetcode.com",
+        "url": "https://leetcode.com",
```
Comment (phallobst) [inline entries/l/leetcode.com.json]:
```suggestion
        "additional-domains": [
            "leetcode.cn"
        ],
```

File: entries/l/leetcode.com.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+        "contact": {
+            "facebook": "LeetCode",
+            "twitter": "LeetCode"
+        },
+        "keywords": [
+            "education"
+        ]
```
Comment (phallobst) [inline entries/l/leetcode.com.json:18]:
```suggestion
        ],
        "regions": [
            "us",
            "cn"
        ]
```

File: img/c/codesignal.com.svg [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  <g clip-path="url(#clip0)">
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M40.226 29.699L39.5767 26.5973L10.8906 22.5411L20.0985 36.6182L21.9873 36.8568L26.8273 44.7901L31.1952 45.3269L40.403 67.0391L49.3748 45.3269L53.8607 44.9691L58.7007 36.8568L60.3534 36.6182L69.7384 22.5411L40.9933 26.5973L40.226 29.699Z" fill="white"></path>
+    <path fill-rule="evenodd" clip-rule="evenodd" d="M35.7982 30.5397L19.3828 27.5516L22.6895 32.6912L32.3734 34.0059L33.3182 37.3526L26.941 36.755L29.421 40.9383L34.5582 41.3566L40.463 59.0462L46.3088 41.3566L51.505 40.9383L53.985 36.755L47.6078 37.3526L48.5526 34.0059L58.2365 32.6912L61.6022 27.5516L45.0097 30.5397L40.4039 48.2293L35.7982 30.5397Z" fill="#004AE1"></path>
+  </g>
+  <defs>
+    <clipPath id="clip0">
+      <rect width="58.8477" height="44.4979" fill="white" transform="translate(10.8906 22.5411)"></rect>
+    </clipPath>
+  </defs>
+</svg>
```
Comment (phallobst) [inline img/c/codesignal.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox=".8 0 79.4 80"><ellipse cx="40.5" cy="40" fill="#004AE1" rx="39.7" ry="40"/><g fill-rule="evenodd" clip-path="url(#a)" clip-rule="evenodd"><path fill="#fff" d="m40.2 29.7-.6-3.1-28.7-4 9.2 14 1.9.3 4.8 7.9 4.4.5L40.4 67l9-21.7 4.5-.3 4.8-8.1 1.7-.3 9.3-14-28.7 4-.8 3.1Z"/><path fill="#004AE1" d="m35.8 30.5-16.4-3 3.3 5.2 9.7 1.3 1 3.4-6.5-.6 2.5 4.1 5.2.5L40.5 59l5.8-17.6 5.2-.5 2.5-4.1-6.4.6 1-3.4 9.6-1.3 3.4-5.1-16.6 3-4.6 17.6-4.6-17.7Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h58.8v44.5H0z" transform="translate(10.9 22.5)"/></clipPath></defs></svg>
```

File: img/l/leetcode.com.svg [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+            <path d="M40.6069914,72.0014117 C37.086019,72.0014117 34.2317068,69.142117 34.2317068,65.6149982 C34.2317068,62.0878794 37.086019,59.2285847 40.6069914,59.2285847 L87.6247154,59.2285847 C91.1456879,59.2285847 94,62.0878794 94,65.6149982 C94,69.142117 91.1456879,72.0014117 87.6247154,72.0014117 L40.6069914,72.0014117 Z" id="Path-2" fill="#B3B3B3"></path>
+            <path d="M49.4124315,2.02335002 C51.8178981,-0.552320454 55.852269,-0.686893945 58.4234511,1.72277172 C60.9946333,4.13243738 61.1289722,8.17385083 58.7235056,10.7495213 L15.9282277,56.5728697 C11.2773659,61.551984 11.7867168,69.5737689 17.1459309,74.8291832 L36.9094236,94.2091099 C39.4255514,96.6764051 39.4686234,100.719828 37.0056277,103.240348 C34.5426319,105.760868 30.5062548,105.804016 27.990127,103.33672 L8.22654289,83.9567041 C-1.92467414,74.0021005 -2.93603527,58.0741402 6.61751533,47.846311 L49.4124315,2.02335002 Z" id="Stroke-3" fill="#000000"></path>
+          </g>
+        </g>
+      </g>
+    </g>
+  </g>
+</svg>
```
Comment (phallobst) [inline img/l/leetcode.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 94 110.92"><g fill="none" fill-rule="evenodd"><path fill="#FFA116" fill-rule="nonzero" d="M67.5 83.07a6.37 6.37 0 0 1 9.02.01 6.4 6.4 0 0 1-.01 9.03l-11.07 11.06a26.7 26.7 0 0 1-37.27.34L8.23 83.96C-1.92 74-2.94 58.07 6.62 47.85l17.8-19.08c9.49-10.15 26.97-11.26 37.8-2.5l16.19 13.1a6.4 6.4 0 0 1-8.01 9.93L54.22 36.22c-5.67-4.59-15.59-3.96-20.48 1.28L15.93 56.57c-4.65 4.98-4.14 13 1.22 18.26L37 94.29a13.96 13.96 0 0 0 19.43-.17l11.08-11.05Z"/><path fill="#B3B3B3" d="M40.6 72a6.38 6.38 0 0 1 0-12.77h47.02a6.38 6.38 0 0 1 0 12.77H40.61Z"/><path fill="#000" d="M49.41 2.02a6.37 6.37 0 0 1 9.01-.3 6.4 6.4 0 0 1 .3 9.03l-42.8 45.82c-4.64 4.98-4.13 13 1.23 18.26L36.9 94.2a6.4 6.4 0 0 1 .1 9.03 6.37 6.37 0 0 1-9.02.1L8.23 83.96C-1.92 74-2.94 58.07 6.62 47.85l42.8-45.83Z"/></g></svg>
```

**Review by phallobst — CHANGES_REQUESTED** [review]

File: img/c/codesignal.com.svg [hunk 7]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox=".8 0 79.4 80"><ellipse cx="40.5" cy="40" fill="#004AE1" rx="39.7" ry="40"/><g fill-rule="evenodd" clip-path="url(#a)" clip-rule="evenodd"><path fill="#fff" d="m40.2 29.7-.6-3.1-28.7-4 9.2 14 1.9.3 4.8 7.9 4.4.5L40.4 67l9-21.7 4.5-.3 4.8-8.1 1.7-.3 9.3-14-28.7 4-.8 3.1Z"/><path fill="#004AE1" d="m35.8 30.5-16.4-3 3.3 5.2 9.7 1.3 1 3.4-6.5-.6 2.5 4.1 5.2.5L40.5 59l5.8-17.6 5.2-.5 2.5-4.1-6.4.6 1-3.4 9.6-1.3 3.4-5.1-16.6 3-4.6 17.6-4.6-17.7Z"/></g><defs><clipPath id="a"><path fill="#fff" d="M0 0h58.8v44.5H0z" transform="translate(10.9 22.5)"/></clipPath></defs></svg>
```
Comment (Carlgo11) [inline img/c/codesignal.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 80 80"><circle cx="40" cy="40" r="40" fill="#004ae1"/><path fill="#fff" d="m10.9 22.54 9.2 14.08 1.89.24 4.84 7.93 4.36.54 9.21 21.7 8.98-21.7 4.48-.36 4.84-8.11 1.65-.24 9.39-14.08-28.75 4.06-.76 3.1-.65-3.1zm8.48 5.01 16.42 2.99 4.6 17.69 4.61-17.7 16.6-2.98-3.37 5.14-9.69 1.32-.94 3.34 6.37-.6-2.47 4.19-5.2.42-5.85 17.69-5.9-17.7-5.14-.41-2.48-4.18 6.38.6-.95-3.35-9.68-1.32z"/></svg>
```

File: entries/l/leetcode.com.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
+{
+    "LeetCode": {
+        "domain": "leetcode.com",
+        "additional-domains": [
+            "leetcode.cn"
+        ],
+        "contact": {
+            "facebook": "LeetCode",
+            "twitter": "LeetCode"
```
Comment (Carlgo11) [inline entries/l/leetcode.com.json]: They advertise this email address on their Facebook page.
```suggestion
            "twitter": "LeetCode",
            "email": "admin@leetcode.com"
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/l/leetcode.com.svg [hunk 9]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 94 110.92"><g fill="none" fill-rule="evenodd"><path fill="#FFA116" fill-rule="nonzero" d="M67.5 83.07a6.37 6.37 0 0 1 9.02.01 6.4 6.4 0 0 1-.01 9.03l-11.07 11.06a26.7 26.7 0 0 1-37.27.34L8.23 83.96C-1.92 74-2.94 58.07 6.62 47.85l17.8-19.08c9.49-10.15 26.97-11.26 37.8-2.5l16.19 13.1a6.4 6.4 0 0 1-8.01 9.93L54.22 36.22c-5.67-4.59-15.59-3.96-20.48 1.28L15.93 56.57c-4.65 4.98-4.14 13 1.22 18.26L37 94.29a13.96 13.96 0 0 0 19.43-.17l11.08-11.05Z"/><path fill="#B3B3B3" d="M40.6 72a6.38 6.38 0 0 1 0-12.77h47.02a6.38 6.38 0 0 1 0 12.77H40.61Z"/><path fill="#000" d="M49.41 2.02a6.37 6.37 0 0 1 9.01-.3 6.4 6.4 0 0 1 .3 9.03l-42.8 45.82c-4.64 4.98-4.13 13 1.23 18.26L36.9 94.2a6.4 6.4 0 0 1 .1 9.03 6.37 6.37 0 0 1-9.02.1L8.23 83.96C-1.92 74-2.94 58.07 6.62 47.85l42.8-45.83Z"/></g></svg>
```
Comment (Carlgo11) [inline img/l/leetcode.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" fill="none" stroke-linecap="round" stroke-width="12.5" viewBox="0 0 94 112"><path stroke="#b3b3b3" d="m40.46 66.52 47.39-.03"/><path stroke="#ffa116" d="M72.35 88.07c-5.09 5.45-9.1 9.15-12.63 12.49a18.47 18.47 0 0 1-11.92 4.94c-4.54.26-9.63-1.1-14.8-5.23-.9-.73-1.82-1.91-2.77-2.79m44.19-52.33S60.6 34 56.83 31.05c-3.76-2.94-16.29-7.28-30.18 5.57"/><path stroke="#000" d="m54.97 6.25-43.98 47.2a18.7 18.7 0 0 0-4.7 13.07c.27 3.35 2.4 9.26 5.47 12.34 0 0 11.7 11.92 20.38 20.6"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by phallobst — APPROVED** [review]

**Pull request merged**